woocommerce/woocommerce-gutenberg-products-block
# Update/typescript checkout shipping

PR adding types to /packages/checkout/shipping directory. 

How to test the changes in this Pull Request:
Checkout branch and make sure code is free of TS warnings errors.
 Testing is limited to smoke test the Cart and Checkout blocks to verify there are no regressions.

Just a note that this PR already typed FormattedMonetaryAmount component. So I just kept my file and ignored the change brought to the file here https://github.com/woocommerce/woocommerce-gutenberg-products-block/commit/aa82c92d5883c09d80aed60291d13661164e3437#diff-d6ebd1dae536334d33ae51da0820ecd0896c24c57a34f5ddaebd0f8b1c98c12cL36
